### PR TITLE
Refactored master to control plane in ha_master.go

### PR DESCRIPTION
**Pull request by mgutierrez98**
#### What type of PR is this?
/kind cleanup
/wg naming
/sig testing
/area test

#### What this PR does / why we need it:
Refactored instances of master to control plane

#### Which issue(s) this PR fixes:
Fixes #

#### Special notes for your reviewer:
references issue #94901

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```docs

```

### CODE REVIEW

File: test/e2e/cloud/gcp/ha_master.go [hunk 1]
```diff
@@ -37,17 +37,17 @@ import (
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
 )
 
-func addMasterReplica(zone string) error {
-	framework.Logf(fmt.Sprintf("Adding a new master replica, zone: %s", zone))
+func addControlPlaneReplica(zone string) error {
+	framework.Logf(fmt.Sprintf("Adding a new control plane replica, zone: %s", zone))
 	_, _, err := framework.RunCmd(path.Join(framework.TestContext.RepoRoot, "hack/e2e-internal/e2e-grow-cluster.sh"), zone, "true", "true", "false")
```
[the remark this answers is not in the document]
Comment (oomichi) [reply test/e2e/cloud/gcp/ha_master.go:42]: That would just add another functions, I don't think that is a better one.

[the remark this answers is not in the document]
Comment (mgutierrez98) [reply test/e2e/cloud/gcp/ha_master.go:42]: understood, shall I close this PR then?

[the remark this answers is not in the document]
Comment (oomichi) [reply test/e2e/cloud/gcp/ha_master.go:42]: At this time, it is fine to close this.
After fixing the above hack/e2e-internal/e2e-grow-cluster.sh you can re-open this again I think.
Sorry for that :-(

**Pull request closed without merging**